base/string_utils.cpp

```diff
@@ -337,6 +337,15 @@ bool EndsWith(std::string const & s1, char const * s2)
   return (s1.compare(n - m, m, s2) == 0);
```

Comment:
Now you can simply call EndsWith(s1, std::string_view(s2)) here

Reply:
Fixed